verify-go.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+set -e
+
+gen_checksum()
+{
+    find ./gen -type f -exec md5sum {} \; | sort -k 2 | md5sum
+}
+
+print_checksum_details()
```

Comment:
This prints checksum for each file separately, might be useful for debugging to see which specific files are out of sync.
If this is to noisy, I may remove this.